### PR TITLE
created api routes for order_products and added apiRouter.use require…

### PR DESCRIPTION
**Pull request by DonnyDunamis**
… in api/index.js for order_products

### CODE REVIEW

File: api/order_products.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    } else {
+      next({
+        name: "ErrorUpdatingOrderProduct",
+        message: "An error occured when trying to update this order.",
+      });
+    }
+  } catch ({ name, message }) {
+    next({ name, message });
+  }
+});
```
Comment (scburkha005) [inline api/order_products.js:40]: i think we can leave out price from our patching, what do u think?

I'm thinking something along the lines of this => the price is initially stored in this table when the product is added to an order, storing the current price of the item in order_products at the time that it's added. this way the price will stay the same even if the price of the product that was added changes afterwards. 

So we'll just use the patch route to update the quantity only probably?

**Review by scburkha005 — CHANGES_REQUESTED** [review]
lmk what you think about this price part, maybe I'm thinking about this wrong or missing some edge case scenario

File: api/order_products.js (hunk 1 above)
Comment (DonnyDunamis) [reply api/order_products.js]: Makes sense... and we can always add it back in if we discover we need it, but I agree with your logic. It jives with previous discussions we've had about price locking. Change made and committed.

**Review by scburkha005 — APPROVED** [review]
great work 👍

**Pull request merged**